src/utils/generator.js

```diff
@@ -459,7 +459,7 @@ export class NameGenerator {
       id = this.prefix + base62.encode(this.uidCounter++);
       if (this.uniqueSuffix.length > 0) id += this.uniqueSuffix;
       if (this.debugNames) {
-        if (debugSuffix) id += "_" + debugSuffix.replace(/[.,:]/g, "_");
+        if (debugSuffix) id += "_" + debugSuffix.replace(/[.,:\(\)\"\'\[\]]/g, "_");
```

Comment:
This type of regex string hurts my eyes, I would put it inside a helper function with meaningful name so that I won't need to read/see it unless I am interested.